in/index.html.mako

```diff
@@ -369,6 +376,17 @@
               <span sort="999999">unavailable</span>
             % endif
           </td>
+          <td class="emr">
+            % if inst['pricing'].get('us-east-1', {}).get("emr", {}):
+              <span sort="${inst['pricing']['us-east-1']["emr"]['ec2']} + ${inst['pricing']['us-east-1']["emr"]['emr']}">
+                ec2: $${inst['pricing']['us-east-1']["emr"]['ec2']}
```

Comment:
Why include the ec2 cost here too? Would the additional emr cost not be sufficient? Or perhaps it should be the sum of both?

Reply:
I wasn't sure if the prices for ec2 where different when used with emr together. But it seams they are the same. I will add only the emr prices and a tooltip.